samples/highcharts/accessibility/population-dashboard/demo.js

```diff
@@ -0,0 +1,550 @@
+        }]
+    }]
+}));
+
+var citiesChart = Highcharts.chart('cities-container', Highcharts.merge(detailsOptions, {
+    title: {
+        text: 'Most populated cities'
+    },
+    accessibility: {
+        screenReaderSection: {
+            beforeChartFormat: '<{headingTagName}>{chartTitle}</{headingTagName}><div>{typeDescription}</div>'
+        },
+        typeDescription: 'Chart showing three circles representing cities, with the circle size representing population size.',
+        point: {
+            descriptionFormatter: function (point) {
+                // Use different thousands sep from default to avoid screen reader
+                // interpreting the numbers as separate.
+                var formattedZ = Highcharts.numberFormat(point.z, 0, '.', ',');
+                return point.name + ', ' + formattedZ + ' population.';
+            }
+        }
+    },
+    chart: {
+        type: 'bubble'
+    },
+    xAxis: {
+        visible: false,
+        minPadding: 0.05,
+        maxPadding: 0.05
+    },
+    colorAxis: {
+        min: 0,
+        minColor: '#F6815A',
+        maxColor: '#DC4D45'
+    },
+    tooltip: {
+        headerFormat: '',
+        pointFormat: '<span style="color:{point.color}">●</span> <span style="font-size: 10px"> {point.name}</span><br/>Population: {point.z}',
+        backgroundColor: 'rgba(255, 255, 255, 0.95)'
+    },
+    series: [{
+        maxSize: 90,
+        minSize: 20,
+        dataLabels: {
+            enabled: true,
+            inside: false,
+            verticalAlign: 'bottom',
+            format: '{point.name}',
+            style: {
+                fontWeight: 'normal',
+                fontSize: '12px'
+            }
+        },
+        data: getTopCities('se')
+    }]
+}));
+
+
+function updateDetails(countryCode, countryName) {
+    historyChart.series[0].setData(
+        populationHistory[countryCode].slice(0), false
+    );
+    historyChart.series[0].activeCountryName = countryName;
+    historyChart.update({
+        title: {
+            text: 'Population history: ' + countryName
+        }
+    });
+    increaseChart.series[0].points[0].update({
+        y: getPopulationIncrease(countryCode)
+    });
+    densityChart.series[0].points[0].update({
+        y: getCountryPopulationDensity(countryCode)
+    });
+    citiesChart.series[0].setData(getTopCities(countryCode));
+}
+
+function clickOnPoint(point) {
+    var curSelected = point.series.currentSelectedPoint;
+    if (curSelected) {
+        curSelected.selectedStatus = '';
+        curSelected.update({
+            color: void 0
+        });
+    }
+    point.series.currentSelectedPoint = point;
+    point.selectedStatus = 'Selected.';
+    point.update({
+        color: {
+            pattern: {
+                path: 'M 0 1.5 L 5 1.5 M 0 4 L 5 4',
+                backgroundColor: '#FDD089',
+                color: '#FF8F99',
+                width: 5,
+                height: 5
+            }
+        }
+    });
```

Comment:
All this should be available natively using `point.select()`. The pattern would then be set in the chart configuration, as the `select` state. So unless there is a bug somewhere, this code is overengineered?

Reply:
It works, programmatically and through UI: https://jsfiddle.net/highcharts/79gydo31/

Reply:
Agreed - The only reason for not using `point.select` was the lack of support for it in the a11y module vs `point.update`, but this is something we should fix now that we don't have a deadline. Also we need to block selecting multiple points by holding down Shift - should we have an API option for this?

Reply:
Yes, we have the `select` event with `e.accumulate`: https://api.highcharts.com/class-reference/Highcharts.PointInteractionEventObject.

We can cancel all select event where the UI passes `e.accumulate = true`: https://jsfiddle.net/highcharts/79gydo31/1/